rocksdb_admin/admin_handler.cpp

```diff
@@ -1063,20 +1097,22 @@ void AdminHandler::async_tm_startMessageIngestion(
   }
 
   int64_t message_count = 0;
+  const auto should_deserialize = meta.should_deserialize_kafka_payload;
```

Comment:
where do we set meta.should_deserialize_kafka_payload?

Will it be simpler if we remove should_deserialize_kafka_payload and use a gflag instead?

Reply:
This will be set in the upload script.

There are already too many gflags and I personally prefer segment level configs to be present in the meta_db.